frontend/src/metabase/components/CodeBlock/utils.ts

```diff
@@ -0,0 +1,119 @@
+import { html } from "@codemirror/lang-html";
+import { javascript } from "@codemirror/lang-javascript";
+import { json } from "@codemirror/lang-json";
+import { python } from "@codemirror/lang-python";
+import { StreamLanguage } from "@codemirror/language";
+import { clojure } from "@codemirror/legacy-modes/mode/clojure";
+import { pug } from "@codemirror/legacy-modes/mode/pug";
+import { ruby } from "@codemirror/legacy-modes/mode/ruby";
+import type { EditorState, Extension } from "@codemirror/state";
+import {
+  Decoration,
+  EditorView,
+  RangeSet,
+  type ReactCodeMirrorRef,
+  StateEffect,
+  StateField,
+} from "@uiw/react-codemirror";
+import { handlebarsLanguage as handlebars } from "@xiechao/codemirror-lang-handlebars";
+import { getNonce } from "get-nonce";
+import { type RefObject, useEffect } from "react";
+
+import S from "./CodeBlock.module.css";
+import type { CodeLanguage } from "./types";
+
+export function getLanguageExtension(language: CodeLanguage): Extension {
+  switch (language) {
+    case "clojure":
+      return StreamLanguage.define(clojure);
+    case "erb":
```

Comment:
do you expect to ever support `erb`? if not can we just set erb to use `html` and we never have to have the `erb` option here?

In fact we already do that for `mustache` https://github.com/metabase/metabase/blob/cc016e3dd4ad3c0a2532d268ca90da787bb02ad9/frontend/src/metabase/public/lib/code.ts#L25-L28

why not do the same for erb 😉

Reply:
There is no ERB plugin for codemirror right now unfortunately, but the example is simple enough for us to just display HTML, IMHO.